environment/src/main/resources/schema/app/20210722121419_CB-13110_Add_the_encryption_Arn_parameter_passed_from_CLI_to_AWS_env_creation_in_Cloudbreak.sql

```diff
@@ -0,0 +1,8 @@
+-- // CB-13110:Add the encryption Arn parameter passed from CLI to AWS env creation in Cloudbreak
+-- Migration SQL that makes the change goes here.
+ALTER TABLE environment_parameters ADD COLUMN IF NOT EXISTS encryption_key_arn text;
```

Comment:
Varchar would be a better type here. Please check for length we use for other ARNs (or CRNs)